README.rst

```diff
@@ -79,7 +79,7 @@ To use this library, you will need to create an ESP32_SPI WifiManager, connected
 
 Native Networking
 =================
-To use this library, with boards that have native networking support, you need to be connected to a network. You will also need to set the current time, as this is used to generate time-based authentication keys. One way to do this is by using the `Adafruit IoT Time Service <https://io.adafruit.com/api/docs/#time>`_ via the `Requests library <https://github.com/adafruit/Adafruit_CircuitPython_Requests/>_` with the following code:
+To use this library, with boards that have native networking support, you need to be connected to a network. You will also need to set the current time, as this is used to generate time-based authentication keys. One way to do this is by using the `Adafruit IoT Time Service <https://io.adafruit.com/api/docs/#time>`_ via the `Adafruit Requests library <https://github.com/adafruit/Adafruit_CircuitPython_Requests>`_ with the following code:
```

Comment:
Please change

> Adafruit IoT Time Service

to 


Adafruit IO Time API

Reply:
I had that originally too, then I went to get the link and it says service not API.

https://io.adafruit.com/api/docs/#time

![image](https://user-images.githubusercontent.com/4002194/151269974-4f7e1c61-1305-439d-b4e3-960db372f867.png)